src/test/kotlin/com/pubnub/contract/membership/step/WhenSteps.kt

```diff
@@ -50,16 +50,16 @@ class WhenSteps(
 
         world.pubnub.setMemberships(
             uuid = membershipState.uuid(),
-            channels = listOf(channelMembershipInput),
+            channels = channelMembershipInput,
         ).sync()?.let {
             world.responseStatus = it.status
-            membershipState.memberships = it.data
+            membershipState.returnedMemberships = it.data
         }
     }
 
     @When("I set the membership for current user")
     fun i_set_the_membershipfor_current_user() {
-        val channelMembershipInput = membershipState.membership.let {
+        val channelMembershipInput = membershipState.memberships.map {
```

Comment:
"setMemberships" takes  channels: List<ChannelMembershipInput>, so maybe instead of 
channelMembershipInput
use
channelMembershipInputList  ?